service/build.gradle

```
@@ -105,8 +109,8 @@ task functional(description: 'Runs the functional tests.') {
 checkstyle {
   maxWarnings = 0
   toolVersion = "8.25"
-  // need to set configDir to rootDir otherwise submodule will use submodule/config/checkstyle
-  configDir = file("$rootDir/config/checkstyle")
+  // need to set configDirectory to rootDir otherwise submodule will use submodule/config/checkstyle
+  configDirectory = file("$rootDir/config/checkstyle")
```

intellij complaining about `Access to 'configDirectory' exceeds its access rights` and `Cannot assign 'File' to 'DirectoryProperty' `
Maybe
```suggestion
  configDirectory.set(file("$rootDir/config/checkstyle"))
```

---

yeah, saw that wasn't sure if the pipeline would play along with `configDirectory.set` as `configDirectory = file("$rootDir/config/checkstyle")` was accepted. As the pipeline did not accept making `def inStrictOrder` static as suggested by IJ

---

ahh fair enough